regression/esbmc/github_65/main.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+
+int *p = (int[]){2, 4}; // creates an unnamed static array of type int[2]
+                        // initializes the array to the values {2, 4}
+                        // creates pointer p to point at the first element of the array
+const float *pc = (const float []){1e0, 1e1, 1e2}; // read-only compound literal
+ 
+int main(void)
+{
+    int n = 2, *p = &n;
```

Comment:
Can you change the value of `n`? When p is first created, the first element of the array is also 2.

Reply:
Sure. Implemented as suggested.